kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExportNamer.kt

```diff
@@ -873,29 +875,16 @@ private inline fun StringBuilder.mangledSequence(crossinline mangle: StringBuild
 
 private fun StringBuilder.mangledBySuffixUnderscores() = this.mangledSequence { append("_") }
 
-private fun ObjCExportMapper.canHaveCommonSubtype(first: ClassDescriptor, second: ClassDescriptor): Boolean {
-    if (first.isSubclassOf(second) || second.isSubclassOf(first)) {
-        return true
-    }
-
-    if (first.isFinalClass || second.isFinalClass) {
-        return false
-    }
-
-    return first.isInterface || second.isInterface
-}
+private fun ObjCExportMapper.getClassDescriptor(member: CallableMemberDescriptor) = getClassIfCategory(member) ?: member.containingDeclaration as? ClassDescriptor
```

Comment:
The name doesn't reflect the meaning. Please make it more specific, e.g. `getEnclosingOrCategoryClass`.

Reply:
I'll remove this function since it was only needed for the refactoring